src/rapidfuzz/process_py.py

```diff
@@ -168,66 +174,66 @@ def extract_iter(
 
 @overload
 def extractOne(
-    query: Sequence[Hashable] | None,
-    choices: Iterable[Sequence[Hashable] | None],
+    query: _StringType | None,
+    choices: Mapping[_KeyType, _StringType | None],
     *,
-    scorer: Callable[..., int | float] = WRatio,
-    processor: Callable[..., Sequence[Hashable]] | None = None,
-    score_cutoff: int | float | None = None,
-    score_hint: int | float | None = None,
+    scorer: Callable[..., _ResultType] = WRatio,
+    processor: Callable[..., _StringType] | None = None,
+    score_cutoff: _ResultType | None = None,
+    score_hint: _ResultType | None = None,
     scorer_kwargs: dict[str, Any] | None = None,
-) -> tuple[Sequence[Hashable], int | float, int] | None: ...
+) -> tuple[_KeyType, _ResultType, _StringType] | None: ...
```

Comment:
The return tuple is always `[choice, result, key]`, so we would want to return `tuple[_StringType, _ResultType, _KeyType]`.

Reply:
I've been testing, and inverting wouldn't reflect the library's behavior 

![image](https://github.com/rapidfuzz/RapidFuzz/assets/63890837/5ae27d4b-4511-4f1f-848e-f97d64f915e1)